MAUI/ImageEditor/keyboard-shortcuts.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Keyboard shortcuts in .NET MAUI Image Editor Control | Syncfusion
+description: Learn here about performing actions using keyboard shortcuts in Syncfusion .NET MAUI Image Editor (SfImageEditor) control.
+platform: maui
+control: SfImageEditor
+documentation: ug
+---
+
+## Keyboard shortcuts in .NET MAUI Image Editor (SfImageEditor)
+
+The [.NET MAUI Image Editor](https://www.syncfusion.com/maui-controls/maui-image-editor) allows the users to navigate through Toolbar items using keyboard shortcuts in Windows and macOS platforms. The below table lists the available shortcuts and the corresponding operations.
+
+N> [ToolbarItemSelected](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ToolbarItemSelected) event triggers for toolbar item tapped.
+
+### Toolbar
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Tab<br/><br/></td><td>Moves selection to the next (LTR) or Previous (RTL) focusable toolbar item. 
+<br/><br/></td></tr>
+<tr>
+<td>
+Shift + Tab<br/><br/></td><td>
+Moves selection to the previous (LTR) or next (RTL) focusable toolbar item. 
+<br/><br/></td></tr>
+<tr>
+<td>
+Enter<br/><br/></td><td>
+It triggers {{'[ToolbarItemSelected](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ToolbarItemSelected)'| markdownify}} event<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Z<br/><br/></td><td>
+Reverse the most recent action performed on the image editor. 
+<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + Y<br/><br/></td><td>
+Restores the last undone action.
+<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + S<br/><br/></td><td>
+Save the current image. 
+<br/><br/></td></tr>
+<tr>
+<td>
+Ctrl + O<br/><br/></td><td>
+Opens a new image file for editing. 
```

Comment:
Opens the image browser to import the new image inside ImageEditor.

Reply:
Modified